tests/unit/api/c/dmx_merger/test_dmx_merger.cpp

```diff
@@ -608,139 +599,469 @@ TEST_F(TestDmxMerger, GetSourceWorks)
   EXPECT_NE(sacn_dmx_merger_get_source(merger_handle_, source_handle_2), nullptr);
 }
 
-TEST_F(TestDmxMerger, UpdateSourceDataMergesLevels)
+TEST_F(TestDmxMergerUpdate, MergesLevels)
```

Comment:
On a few of of these tests, you could create variants that do the update operations in a different order to make sure you get the same result.  For example, you could do source 2 first, or interleave setting their priorities and levels.

Reply:
https://github.com/ETCLabs/sACN/pull/29/commits/b74e25cecfecd7383db78cfd3d2f6de98a49c9f6